M2/Macaulay2/packages/Macaulay2Doc/doc5.m2

```diff
@@ -616,11 +620,17 @@ document {
      Key => 1 : setRandomSeed,
      Usage => "setRandomSeed()",
      Consequences => {
-	  {"Initializes the random number generator to a fixed state, identical to the
-	       initial state in version 1.2 and earlier of Macaulay2.  After version 1.2,
-	       the random number seed is initially set to a number that depends on the current date, 
-	       time (in seconds), and process id, except for the running of examples and tests
-	       in packages, where it is always initialized to 0."}
+	  {"Initializes the random number generator to a certain fixed state
+	       determined at the beginning of the current Macaulay2 session. In
```

Comment:
This gives the wrong impression about what "setRandomSeed()" does.  It is provided for backward
compatibility to set the seed as it was initially set in versions 1.2 and earlier.  It is not what is used
to determine the seed at the beginning of a Macaulay2 session.